frameworks/sveltekit.md

```diff
@@ -117,6 +117,24 @@ const config = {
 export default config
 ```
 
+In addition to the configuration above, it's necessary to explain SvelteKit on how to handle and insert the webmanifest into the generated source code. Currently the easiest way to do this, is to add a layout to your kit project.
+
+```svelte
+<!-- src/routes/+layout.svelte -->
+<script>
+  import { pwaInfo } from 'virtual:pwa-info'; 
+
+  let webManifestLink;
+  $: webManifestLink = pwaInfo ? pwaInfo.webManifest.linkTag : '' 
+</script> 
+  
+<svelte:head> 
+ 	{@html webManifestLink} 
+</svelte:head> 
+```
+
+Please see [here](/frameworks/#accessing-pwa-info) to learn more about the virtually exposed module `pwa-info`.
```

Comment:
don't use here, replace it with  `[virtual:pwa-info](/frameworks/#accessing-pwa-info)`